libs/api/domains/directorate-of-labour/src/lib/directorate-of-labour.resolver.ts

```diff
@@ -68,4 +69,13 @@ export class DirectorateOfLabourResolver {
   async getPrivatePensionFunds(): Promise<PensionFund[] | null> {
     return this.directorateOfLabourService.getPrivatePensionFunds()
   }
+
+  @Query(() => ParentalLeavePregnancyStatus, { nullable: true })
+  async getParentalLeavePregnancyStatus(
+    @CurrentUser() user: User,
```

Comment:
Is that working? On my branch, I needed to add back the `@UseGuards(IdsAuthGuard, ScopesGuard)` on top of the file to be able to get the user through the `@CurrentUser` decorator

Reply:
Well spotted! Thanks.